addons/website/static/src/snippets/s_facebook_page/000.js

```diff
@@ -24,10 +22,39 @@ const FacebookPageWidget = publicWidget.Widget.extend({
             params.href = `https://www.facebook.com/${params.id}`;
         }
         delete params.id;
+
+        this._renderIframe(params);
+
+        this.resizeObserver = new ResizeObserver(debounce(this._renderIframe.bind(this, params), 100));
```

Comment:
I think `throttle` sould be more appropriate here to avoid flickering when resizing the window.
I even think that any of these (`debounce` or `throttle` are really necessary actually)

Reply:
Hello @bvr-odoo 
As we are rendering the iframe on resizing, it will trigger an event on every instance of time to render. So, Debounce will delay the method to trigger for about a minimal amount of time on every last event call. In the render function, we are fetching a URL that takes time to render and resize the iframe.

While in throttling, the rendering method would still be triggered at regular intervals during the resize process. This means that even if the resize events occur very frequently, the iframe would still be rendered periodically, leading to multiple renders while resizing is still in progress.

Reply:
> Hello @bvr-odoo As we are rendering the iframe on resizing, it will trigger an event on every instance of time to render. So, Debounce will delay the method to trigger for about a minimal amount of time on every last event call. In the render function, we are fetching a URL that takes time to render and resize the iframe.
> 
> While in throttling, the rendering method would still be triggered at regular intervals during the resize process. This means that even if the resize events occur very frequently, the iframe would still be rendered periodically, leading to multiple renders while resizing is still in progress.

@paru-odoo  Alright, let's leave it like that. Anyway, a resize event usually only triggers once (e.g. screen rotation on mobile). Let me know when the PR is ready for review again. Thanks.